### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by codedit334**
## Hi Reviewer 👋,

In this PR:
- Added SX elements to your project for missing parts.
- Final design matches Zeplin's design.
- Used the CSS rules detailed in Zeplin to style the website.

# 🤝

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -2,35 +2,44 @@ import React from 'react';
...
     <div className="book" data-id={id}>
-      <div className="book-title">
-        Title:
-        {title}
+      <div className="book-wrapper">
+        <div className="book-category">
+          {category}
+        </div>
+        <div className="book-title">
+          {/* <span className="book-space">Title:</span> */}
```
Comment (huseyinbiyik) [inline src/components/Book.js]: Please don't leave any uncommented code in your all codebase. In this way, you can keep your codebase clean, organized, and professional. If you need a specific line of code, you can use git checkout `
<commitnumber>`  .

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @codedit334 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝

## Required Changes ♻️
Screenshot 2023-04-05 at 9 01 39 PM

You did a great job adding your design taste to this project. But unfortunately, we expect you to code the Zeplin design **exactly** the same. 
Screenshot 2023-04-05 at 8 48 24 PM Since Zeplin gives you all the CSS values, all you need to do is to implement the design with  [HTML&CSS Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):  , considering the rules below.
No repeating of the same rule
No ID selectors for multiple elements
No !important overusing
No inefficient selectors
No inline styling
Screenshot 2023-04-05 at 8 57 40 PM

-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML" 

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codedit334** [comment]
Hi @huseyinbiyik, Thank you for taking the time to review my work,
I made sure the overall layout is similar to the one on Zeplin...
By 'Match the Zeplin design' do You mean similar colors or similar CSS values?

**Comment by huseyinbiyik** [comment]
Hey @codedit334 , This time, it needs to be exactly the same as Zeplin's design. Not the best example but please check this [one](https://deploy-preview-5--shimmering-kangaroo-bc7705.netlify.app/Books).

**Comment by codedit334** [comment]
Thank you @huseyinbiyik

**Review by huseyinbiyik — APPROVED** [review]
Hi @codedit334 again,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- The design looks identical. 👌 
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝
- Great implementation of the requested changes 🕸️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**